assets/report_template.Rmd

```diff
@@ -0,0 +1,1175 @@
+    flag_single_end: FALSE
+    barplot: FALSE
+    abundance_tables: FALSE
+    alpha_rarefaction: FALSE
+    ancom: FALSE
+    trunclenf: ""
+    trunclenr: ""
+    max_ee: ""
+    trunc_qmin: FALSE
+    trunc_rmin: ""
+    dada_sample_inference: ""
+    filter_ssu: FALSE
+    min_len_asv: ""
+    max_len_asv: ""
+    cut_its: FALSE
+    dada2_ref_tax_title: ""
+    qiime2_ref_tax_title: ""
+    sintax_ref_tax_title: ""
+    exclude_taxa: ""
+    min_frequency: ""
+    min_samples: ""
+    qiime2_filtertaxa: ""
+    val_used_taxonomy: FALSE
+    metadata_category_barplot: FALSE
+    qiime_adonis_formula: FALSE
+
+    # file paths
+    mqc_plot: FALSE
+    ca_sum_path: FALSE
+    dada_filtntrim_args: FALSE
+    dada_qc_f_path: FALSE
+    dada_qc_r_path: ""
+    dada_pp_qc_f_path: ""
+    dada_pp_qc_r_path: ""
+    dada_err_path: FALSE
+    dada_err_run: ""
+    asv_table_path: FALSE
+    path_asv_fa: FALSE
+    path_dada2_tab: FALSE
+    dada_stats_path: FALSE
+    path_barrnap_sum: FALSE
+    filter_ssu_stats: ""
+    filter_ssu_asv: ""
+    filter_len_asv: FALSE
+    filter_len_asv_len_orig: FALSE
+    filter_codons: FALSE
+    stop_codons: ""
+    itsx_cutasv_summary: ""
+    cut_dada_ref_taxonomy: FALSE
+    dada2_taxonomy: FALSE
+    sintax_taxonomy: FALSE
+    pplace_taxonomy: FALSE
+    pplace_heattree: ""
+    qiime2_taxonomy: FALSE
+    filter_stats_tsv: FALSE
+    diversity_indices_depth: ""
+    diversity_indices_beta: FALSE
+    diversity_indices_adonis: ""
+    picrust_pathways: FALSE
+---
+
+<!-- Load libraries -->
+
+```{r libraries, include=FALSE}
+library("dplyr")
+library("ggplot2")
+library("knitr")
+library("DT")
+library("formattable")
+library("purrr")
+```
+
+<!-- set notebook defaults -->
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE) # echo is set in differentialabundance v1.2.0 to TRUE
+```
+
+<!-- Include the CSS and set the logo -->
+
+```{r, echo=FALSE}
+htmltools::includeCSS(params$css)
+```
+
+```{r results="asis", echo=FALSE}
+cat(paste0("
+<style>
+#TOC {
+   background-image: url(\"", knitr::image_uri(params$logo), "\");
+}
+</style>
+"))
+```
+
+```{r}
+if ( endsWith( params$workflow_manifest_version, "dev") ) {
+    ampliseq_version = paste0("version ", params$workflow_manifest_version, ", revision ", params$workflow_scriptid)
+} else {
+    ampliseq_version = paste0("version ",params$workflow_manifest_version)
+}
+report_title <- "Summary of analysis results"
+report_subtitle <- paste0('nf-core/ampliseq workflow ', ampliseq_version)
+```
+
+---
+title:  "<img src=\"`r file.path(params$input_dir, params$logo)`\" style=\"float: left;\"/>`r report_title`"
+subtitle: `r report_subtitle`
+---
+
+# Abstract
+
+The bioinformatics analysis pipeline [nfcore/ampliseq](https://nf-co.re/ampliseq) is used for amplicon sequencing,
+supporting denoising of any amplicon and supports a variety of taxonomic databases for taxonomic assignment of 16S, ITS, CO1 and 18S amplicons.
```

Comment:
```suggestion
The bioinformatics analysis pipeline [nfcore/ampliseq](https://nf-co.re/ampliseq) is used for amplicon sequencing; it supports denoising of any amplicon and a variety of taxonomic databases for taxonomic assignment of 16S, ITS, CO1 and 18S amplicons.
```

Reply:
May I asked why you propose to removed the newline? As far as I can see the newline is ignored in the rendered html. The idea why I had it in there was to make the code more readable by adding the line break.